src/query/functions/src/scalars/timestamp/src/datetime.rs

```diff
@@ -698,24 +597,15 @@ fn register_to_string(registry: &mut FunctionRegistry) {
         |_, _, _| FunctionDomain::MayThrow,
         vectorize_with_builder_2_arg::<TimestampType, StringType, NullableType<StringType>>(
             |micros, format, output, ctx| {
-                if format.is_empty() {
-                    output.push_null();
-                } else {
-                    let items = StrftimeItems::new(format);
-                    if items.clone().any(|item| matches!(item, Item::Error)) {
-                        ctx.set_error(output.len(), "Invalid format string".to_string());
+                let ts = micros.to_timestamp(ctx.func_ctx.jiff_tz.clone());
```

Comment:
The name of jiff_tz is a different with Chrono tz. But now we will drop Chrono in function crate, so maybe we can rename it as tz